tests/glucose_tests.py

```diff
@@ -11,7 +11,9 @@ def get_file_at_path(path):
 
 class CleanTestCase(unittest.TestCase):
     def test_clean_values(self):
```

Comment:
Let's make two separate functions instead: `test_clean_values_mm` and `test_clean_values_dex`